pkg/ebpftracer/helpers/helpers.go

```diff
@@ -0,0 +1,20 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/cilium/ebpf"
+)
+
+func SetVariable(spec *ebpf.CollectionSpec, variable string, value any) error {
```

Comment:
Nitpick: I would keep this as internal function under ebpftracer package. Having such helpers package is not a good practice and tends to eventually have some random functions.